src/main/java/uk/gov/digital/ho/hocs/casework/domain/model/StageWithCaseData.java

```diff
@@ -124,40 +125,11 @@ public StageWithCaseData(UUID caseUUID, String stageType, UUID teamUUID, UUID us
         this.caseUUID = caseUUID;
         this.stageType = stageType;
         this.transitionNoteUUID = transitionNoteUUID;
-        completed = Boolean.FALSE;
+        this.completed = Boolean.FALSE;
         setTeam(teamUUID);
-        setUser(userUUID);
-    }
-
-    public void setDeadline(LocalDate deadline) {
-        this.deadline = deadline;
-    }
-
-    public void setDeadlineWarning(LocalDate deadlineWarning){
-        this.deadlineWarning = deadlineWarning;
-    }
-
-    public void setTransitionNote(UUID transitionNoteUUID) {
-        this.transitionNoteUUID = transitionNoteUUID;
-    }
-
-    public void setTeam(UUID teamUUID) {
-        this.teamUUID = teamUUID;
-        this.userUUID = null;
```

Comment:
Is this something we still want to keep?

Reply:
its already defined in the base class and then defined again here so it is safe to delete this one - the behaviour is important because it ensures that a case isn't still allocated to the last user when it lands in a team.